IBM/ibm-common-service-operator
# [WIP] Create keycloak BR process

For 4.4, develop and test a backup and restore process for keycloak/edb. This is based on the existing process we have for Zen v5's edb. The related files for zen are located in the same directory.

How to test:
- Edit and apply the necessary files (look for any values in `<>` in any files with `keycloak` in the name)
- Can either run velero backup and restore or manually run the scripts from inside the keycloak-backup deployment pod
- verify on same cluster and cluster to cluster that expected data has been restore and functionality has not been impacted
